CMakeLists.txt

```diff
@@ -49,21 +49,55 @@ include(cmake/O3DEJson.cmake)
 # Subdirectory processing
 ################################################################################
 
+function(add_engine_gem_json_external_subdirectories gem_path)
```

Comment:
nit: Should add a comment or description here that this function is building up the ```LY_EXTERNAL_SUBDIRS``` global property

Reply:
Ok.